src/core/index.js

```diff
@@ -27,7 +26,8 @@ function IPFS (repoInstance) {
     throw new Error('Must be instantiated with new')
   }
 
-  if (!(repoInstance instanceof IPFSRepo)) {
+  if (typeof repoInstance === 'string' ||
+      repoInstance === undefined) {
```

Comment:
I've noticed that on Travis, the "isDaemonOn" always yielded `true`, making the 'cli offline' tests work online. Checking if this was the issue. @nginnever @dignifiedquire